src/teleop/jetson/jetson_teleop.py

```diff
@@ -189,18 +195,30 @@ def ra_open_loop_control(self, msg: Joy) -> None:
         right_trigger = raw_right_trigger if raw_right_trigger > 0 else 0
-
-        self.ra_cmd.velocity = [
-            self.ra_config["joint_a"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x"),
-            self.ra_config["joint_b"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_y"),
-            self.ra_config["joint_c"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_y"),
-            self.ra_config["joint_d"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_x"),
-            self.ra_config["joint_e"]["multiplier"] * (right_trigger - left_trigger),
-            self.ra_config["joint_f"]["multiplier"]
-            * self.filter_xbox_button(msg.buttons, "right_bumper", "left_bumper"),
-            self.ra_config["finger"]["multiplier"] * self.filter_xbox_button(msg.buttons, "y", "a"),
-            self.ra_config["gripper"]["multiplier"] * self.filter_xbox_button(msg.buttons, "b", "x"),
-        ]
+        if self.ra_slow_mode == False:
+            self.ra_cmd.velocity = [
+                self.ra_config["joint_a"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x"),
+                self.ra_config["joint_b"]["multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_y"),
+                self.ra_config["joint_c"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_y"),
+                self.ra_config["joint_d"]["multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_x"),
+                self.ra_config["joint_e"]["multiplier"] * (right_trigger - left_trigger),
+                self.ra_config["joint_f"]["multiplier"]
+                * self.filter_xbox_button(msg.buttons, "right_bumper", "left_bumper"),
+                self.ra_config["finger"]["multiplier"] * self.filter_xbox_button(msg.buttons, "y", "a"),
+                self.ra_config["gripper"]["multiplier"] * self.filter_xbox_button(msg.buttons, "b", "x"),
+            ]
+        else:
+             self.ra_cmd.velocity = [
+                self.ra_config["joint_a"]["slow_mode_multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_x"),
+                self.ra_config["joint_b"]["slow_mode_multiplier"] * self.filter_xbox_axis(msg.axes, "left_js_y"),
+                self.ra_config["joint_c"]["slow_mode_multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_y"),
+                self.ra_config["joint_d"]["slow_mode_multiplier"] * self.filter_xbox_axis(msg.axes, "right_js_x"),
+                self.ra_config["joint_e"]["slow_mode_multiplier"] * (right_trigger - left_trigger),
+                self.ra_config["joint_f"]["slow_mode_multiplier"]
+                * self.filter_xbox_button(msg.buttons, "right_bumper", "left_bumper"),
+                self.ra_config["finger"]["slow_mode_multiplier"] * self.filter_xbox_button(msg.buttons, "y", "a"),
+                self.ra_config["gripper"]["slow_mode_multiplier"] * self.filter_xbox_button(msg.buttons, "b", "x"),
+            ]
```

Comment:
I think it may be better to separate out the inversion multiplier from the slow mode multiplier. AKA the slow mode multipliers should all be positive and we should always use the normal inversion multiplier.

Then after applying the inversion multiplier, we can just say if slow mode, for each joint, multiply by slow mode multiplier. This may also make it possible to use a for loop to iterate over the strings "joint_a", ... "gripper" and avoid this code duplication

Reply:
Yea I think it would be better to first just calculate as if there was no slow mode option, and then do a check right after if slow mode to just multiply by the slow mode multiplier.

Reply:
I agree with this, this has been something that has confused people in the past.